exporter/datadogexporter/factory.go

```diff
@@ -68,6 +74,35 @@ func (f *factory) SourceProvider(set component.TelemetrySettings, configHostname
 	return f.sourceProvider, f.providerErr
 }
 
+// Reporter builds and returns an *inframetadata.Reporter.
+func (f *factory) Reporter(params exporter.CreateSettings, pcfg hostmetadata.PusherConfig) (*inframetadata.Reporter, error) {
+	f.onceReporter.Do(func() {
+		pusher := hostmetadata.NewPusher(params, pcfg)
+		period := 30 * time.Minute
```

Comment:
Is this meant to be fixed or can it be configurable?

Reply:
It's meant to be fixed, should I move it to a constant?

Reply:
Yeah that makes more sense

Reply:
Moved to file-level constant